### PR TITLE
Changes to GUI + translation keys

### PR DESCRIPTION
**Pull request by NerdyPuzzle**
Changed the GUI to look more like MCreator's elements + renamed some translation keys for better clarity.

image

### CODE REVIEW

**Comment by Azzy380** [comment]
Thank you for fixing my terrible English and Gui! 
But can you change `boarder` in lang? (It hurts even my eyes) :>

**Review by Azzy380 — CHANGES_REQUESTED** [review]
Thank you for fixing my GUI and English!
But can you change `boarder` to `border` in lang?

File: src/main/resources/lang/texts.properties [hunk 1]
```diff
@@ -2,11 +2,12 @@ elementgui.attribute.page.new=Attribute
...
-elementgui.attribute.description=Name of attribute to show in game
+elementgui.attribute.description=In-game attribute name
 elementgui.attribute.defaultValue=Default Value
-elementgui.attribute.minValue=Minimal Value
-elementgui.attribute.maxValue=Maximal Value
-elementgui.attribute.isPersistent=Force stay after death
+elementgui.attribute.minValue=Minimum Value
+elementgui.attribute.maxValue=Maximum Value
+elementgui.attribute.isPersistent=Remain after death
+elementgui.attribute.boarder=Attribute Properties
```
Comment (Azzy380) [inline src/main/resources/lang/texts.properties]: `boarder` hurts my already bad English :<

Comment (NerdyPuzzle) [reply src/main/resources/lang/texts.properties]: Oops!

**Review by Azzy380 — APPROVED** [review]
nice :3

**Pull request merged**